connector/connect/src/main/protobuf/spark/connect/base.proto

```diff
@@ -83,7 +83,6 @@ message Response {
     int64 uncompressed_bytes = 2;
```

Comment:
@grundprinzip do you know why we need this?

Reply:
I think a pre-mature optimization in my head wanted to put some streaming compression on it. We can probably just get rid of them for now.

Reply:
@zhengruifeng let;s get rid of this, then we don;t need to estimate the size, and would be easier to deduplicate the codes.